spine-csharp/src/Animation.cs

```diff
@@ -249,13 +249,13 @@ public RotateTimeline (int frameCount)
 			}
 
 			// Interpolate between the previous frame and the current frame.
-			int frameIndex = Animation.binarySearch(frames, time, 2);
-			float prevFrameValue = frames[frameIndex - 1];
-			float frameTime = frames[frameIndex];
-			float percent = 1 - (time - frameTime) / (frames[frameIndex + PREV_FRAME_TIME] - frameTime);
-			percent = GetCurvePercent((frameIndex >> 1) - 1, percent < 0 ? 0 : (percent > 1 ? 1 : percent));
+			int frame = Animation.binarySearch(frames, time, 2);
```

Comment:
When adding TransformConstraintTimeline and I had a `FRAME_PREV_TRANSLATE_MIX`, I decided the `FRAME_` prefix wasn't helpful and made everything super long. I updated all the other timelines to keep things consistent. Spring cleaning!